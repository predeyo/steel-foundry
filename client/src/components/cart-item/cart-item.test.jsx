import React from "react";
import { shallow } from "enzyme";
import CartItem from "./cart-item.component.jsx";

describe("CartItem component", () => {
  let wrapper;
  let mockName = "Purple leggings";
  let mockPrice = 104;
  let mockImageUrl = "www.imagetest.com";
  let mockQuantity = 99;
  let mockProps;
  beforeEach(() => {
    mockProps = {
      item: {
        name: mockName,
        price: mockPrice,
        imageUrl: mockImageUrl,
        quantity: mockQuantity
      }
    };
    wrapper = shallow(<CartItem {...mockProps} />);
  });

  it("should render CartItem component", () => {
    expect(wrapper.debug()).toMatchSnapshot();
  });
});
