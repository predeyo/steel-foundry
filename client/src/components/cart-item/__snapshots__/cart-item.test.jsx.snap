// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CartItem component should render CartItem component 1`] = `
"<div className=\\"cart-item\\">
  <img src=\\"www.imagetest.com\\" alt=\\"item\\" />
  <div className=\\"item-details\\">
    <span className=\\"name\\">
      Purple leggings
    </span>
    <span className=\\"price\\">
      99
       x $
      104
    </span>
  </div>
</div>"
`;
